content/en/user-guide/lambda-tools/hot-reloading/index.md

```diff
@@ -23,19 +23,6 @@ The configuration `BUCKET_MARKER_LOCAL` is still supported.
 More information about the new Lambda provider is available under [Lambda providers]({{< ref "user-guide/aws/lambda" >}}).
 {{< /alert >}}
 
-## Covered Topics
-
-* [Hot Reloading Behavior](#hot-reloading-behavior)
-* [Application Configuration Examples](#application-configuration-examples):
-  * [Hot reloading for JVM Lambdas](#hot-reloading-for-jvm-lambdas)
-  * [Hot reloading for Python Lambdas](#hot-reloading-for-python-lambdas)
-  * [Hot reloading for TypeScript Lambdas](#hot-reloading-for-typescript-lambdas)
-* [Deployment Configuration Examples](#deployment-configuration-examples):
-  * [Serverless Framework Configuration](#serverless-framework-configuration)
-  * [AWS Cloud Development Kit (CDK) Configuration](#aws-cloud-development-kit-cdk-configuration)
-  * [Terraform Configuration](#terraform-configuration)
-* [Useful Links](#useful-links)
```

Comment:
Why are we removing this?

Reply:
It's duplicate and not necessary.
![image](https://github.com/localstack/docs/assets/97876429/96e398ea-7618-44c1-9667-629f7846b617)